helm/cluster-openstack/templates/_cluster_class.tpl

```diff
@@ -16,7 +16,7 @@ spec:
       ref:
         apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
         kind: OpenStackMachineTemplate
-        name: {{ include "resource.default.name" . }}-{{ .Values.controlPlane.class }}
+        name: {{ include "resource.default.name" . }}-control-plane
```

Comment:
Does a cluster upgrade like this work? Needs to be tested for sure.

Reply:
If it doesn't, it's a big problem. I will test an upgrade from the previous version and report back.

Reply:
I'm sorry to report that cluster upgrades won't work because `OpenStackMachineTemplate` is immutable. This is not unique to this PR and would affect anything that changes the structure of a cluster. I have a PR to fix this using template rotation here https://github.com/giantswarm/cluster-openstack/pull/25. We're going to group this in with the OIDC changes and require that existing clusters are re-created to get this set of changes.